artemis-server/src/main/java/org/apache/activemq/artemis/core/server/management/impl/ManagementServiceImpl.java

```diff
@@ -692,16 +644,16 @@ public SimpleString getManagementNotificationAddress() {
       return managementNotificationAddress;
    }
 
-   // ActiveMQComponent implementation -----------------------------
-
    @Override
-   public void start() throws Exception {
+   public synchronized void start() throws Exception {
+      if (started) {
```

Comment:
This is a lot less protection than before for concurrent calls to start since you aren't using an atomic operation to compare and set so you can get two concurrent start calls competing in here.  Would depend a lot on whether the call is idempotent.

Reply:
Won't the fact that `start` is `synchronized` stop concurrent calls?

Reply:
You are right, I got glassy eyed reading all the other removes of synchronized and read that backwards.  Ignore me.